.bin/srclib-csharp

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+BIN_DIR="`dirname \"$BASH_SOURCE\"`"
+ROOT_DIR="`dirname \"$BIN_DIR\"`"
+PROJ_DIR="$ROOT_DIR/Srclib.Nuget"
+#echo "$PROJ_DIR"
+
+#echo dnx -p "$PROJ_DIR/project.json" run $*
+dnx -p "$PROJ_DIR" run $*
```

Comment:
For ease of development, it's nice to have those commands in a Makefile. I think most of the other toolchains do that. Then, the function in `srclib/cli/toolchain_cmd.go` invokes `make dep`.